src/test/scala/edu/berkeley/ce/rockslicing/InputProcessingSpec.scala

```diff
@@ -5,169 +5,261 @@ import org.scalatest.FunSuite
 import scala.io.Source
 
 class InputProcessingSpec extends FunSuite {
+
+  def compareInputFaces(face1: InputFace, face2: InputFace): Boolean = {
```

Comment:
How about simplifying the body of this function to:
```scala
(face1.strike == face2.strike &&
       face1.dip == face2.dip &&
       (face1.pointInPlane sameElements face2.pointInPlane) &&
 +     face1.phi == face2.phi &&
       face1.cohesion == face2.cohesion
```

Should avoid some unnecessary code and have the same effect.